### PR TITLE
Mark Annotation and FixedPoint as experimental

### PR DESCRIPTION
**Pull request by chick**
Moved interfaces into experimental package
Fix tests and other references to these constructs

### CODE REVIEW

File: src/test/scala/chiselTests/AnnotatingDiamondSpec.scala [hunk 1]
```diff
@@ -3,6 +3,7 @@
 package chiselTests
 
 import chisel3._
+import chisel3.experimental._
```
Comment (ducky64) [inline src/test/scala/chiselTests/AnnotatingDiamondSpec.scala]: Consider just importing the relevant parts of experimental.

**Comment by stevobailey** [comment]
Why? I seemed to have missed a discussion about this.

**Comment by chick** [comment]
This came out of a *cleanup the chisel api* meeting.  I probably jumped the gun on approving it, I was thinking the cleanup would be just in dsptools.

**Comment by chick** [comment]
Could be task for next hacking session, shouldn't take very long to fix imports

**Comment by stevobailey** [comment]
Okay, so this isn't something that will be moved out of experimental anytime soon? If not, then yeah we can go ahead and add the import statements everywhere on Monday

**Pull request merged**